src/main/java/oss/fosslight/common/CoConstDef.java

```diff
@@ -818,19 +819,83 @@ public enum CAMPAIGN_ROUTE {
+		HTTP_CLIENT_TIMEOUT(HttpURLConnection.HTTP_CLIENT_TIMEOUT, "HTTP_CLIENT_TIMEOUT"),
+		HTTP_CONFLICT(HttpURLConnection.HTTP_CONFLICT, "HTTP_CONFLICT"),
+		HTTP_CREATED(HttpURLConnection.HTTP_CREATED, "HTTP_CREATED"),
+		HTTP_ENTITY_TOO_LARGE(HttpURLConnection.HTTP_ENTITY_TOO_LARGE, "HTTP_ENTITY_TOO_LARGE"),
+		HTTP_FORBIDDEN(HttpURLConnection.HTTP_FORBIDDEN, "HTTP_FORBIDDEN"),
+		HTTP_GATEWAY_TIMEOUT(HttpURLConnection.HTTP_GATEWAY_TIMEOUT, "HTTP_GATEWAY_TIMEOUT"),
+		HTTP_GONE(HttpURLConnection.HTTP_GONE, "HTTP_GONE"),
+		HTTP_INTERNAL_ERROR(HttpURLConnection.HTTP_INTERNAL_ERROR, "HTTP_INTERNAL_ERROR"),
+		HTTP_LENGTH_REQUIRED(HttpURLConnection.HTTP_LENGTH_REQUIRED, "HTTP_LENGTH_REQUIRED"),
+		HTTP_MOVED_PERM(HttpURLConnection.HTTP_MOVED_PERM, "HTTP_MOVED_PERM"),
+		HTTP_MOVED_TEMP(HttpURLConnection.HTTP_MOVED_TEMP, "HTTP_MOVED_TEMP"),
+		HTTP_MULT_CHOICE(HttpURLConnection.HTTP_MULT_CHOICE, "HTTP_MULT_CHOICE"),
+		HTTP_NO_CONTENT(HttpURLConnection.HTTP_NO_CONTENT, "HTTP_NO_CONTENT"),
+		HTTP_NOT_ACCEPTABLE(HttpURLConnection.HTTP_NOT_ACCEPTABLE, "HTTP_NOT_ACCEPTABLE"),
+		HTTP_NOT_AUTHORITATIVE(HttpURLConnection.HTTP_NOT_AUTHORITATIVE, "HTTP_NOT_AUTHORITATIVE"),
+		HTTP_NOT_FOUND(HttpURLConnection.HTTP_NOT_FOUND, "HTTP_NOT_FOUND"),
+		HTTP_NOT_IMPLEMENTED(HttpURLConnection.HTTP_NOT_IMPLEMENTED, "HTTP_NOT_IMPLEMENTED"),
+		HTTP_NOT_MODIFIED(HttpURLConnection.HTTP_NOT_MODIFIED, "HTTP_NOT_MODIFIED"),
+		HTTP_OK(HttpURLConnection.HTTP_OK, "HTTP_OK"),
+		HTTP_PARTIAL(HttpURLConnection.HTTP_PARTIAL, "HTTP_PARTIAL"),
+		HTTP_PAYMENT_REQUIRED(HttpURLConnection.HTTP_PAYMENT_REQUIRED, "HTTP_PAYMENT_REQUIRED"),
+		HTTP_PRECON_FAILED(HttpURLConnection.HTTP_PRECON_FAILED, "HTTP_PRECON_FAILED"),
+		HTTP_PROXY_AUTH(HttpURLConnection.HTTP_PROXY_AUTH, "HTTP_PROXY_AUTH"),
+		HTTP_REQ_TOO_LONG(HttpURLConnection.HTTP_REQ_TOO_LONG, "HTTP_REQ_TOO_LONG"),
+		HTTP_RESET(HttpURLConnection.HTTP_RESET, "HTTP_RESET"),
+		HTTP_SEE_OTHER(HttpURLConnection.HTTP_SEE_OTHER, "HTTP_SEE_OTHER"),
+		HTTP_UNAUTHORIZED(HttpURLConnection.HTTP_UNAUTHORIZED, "HTTP_UNAUTHORIZED"),
+		HTTP_UNAVAILABLE(HttpURLConnection.HTTP_UNAVAILABLE, "HTTP_UNAVAILABLE"),
+		HTTP_UNSUPPORTED_TYPE(HttpURLConnection.HTTP_UNSUPPORTED_TYPE, "HTTP_UNSUPPORTED_TYPE"),
+		HTTP_USE_PROXY(HttpURLConnection.HTTP_USE_PROXY, "HTTP_USE_PROXY"),
+		HTTP_VERSION(HttpURLConnection.HTTP_VERSION, "HTTP_VERSION");
+
+		private final int code;
+		private final String message;
+
+		HttpStatusCode(int code, String message) {
+			this.code = code;
+			this.message = message;
+		}
+
+		public int getCode() {
+			return code;
+		}
+
+		public String getMessage() {
+			return message;
+		}
+
+		public static String httpCodePrint(int code) {
+			for (HttpStatusCode status : HttpStatusCode.values()) {
+				if (status.getCode() == code) {
+					return status.getMessage();
+				}
+			}
+			throw new IllegalArgumentException("Unknown HTTP status code: " + code);
+		}
+	}

```

Comment:
@parkcoldroad Nice try! I think the `message` could be replaced with the enum's built-in method, `name()`.
```suggestion
		private final int code;

		HttpStatusCode(int code) {
			this.code = code;
		}

		private int getCode() {
			return code;
		}

		public static String httpCodePrint(int code) {
			for (HttpStatusCode status : HttpStatusCode.values()) {
				if (status.getCode() == code) {
					return status.name();
				}
			}
			return code + "";
		}
	}

```

Reply:
Thank you for your advice. I'll apply right away!